alkira/resource_alkira_connector_azure_vnet.go

```diff
@@ -243,18 +248,21 @@ func resourceAlkiraConnectorAzureVnet() *schema.Resource {
 				Type:        schema.TypeString,
 				Optional:    true,
 			},
+			"cxp_peering_gateway_id": {
```

Comment:
I think this field should be consistent with resource name, which is `peering_gateway_cxp_id`